src/templates/en/2020/index.html

```diff
@@ -22,3 +22,6 @@
 
 <a href="https://github.com/HTTPArchive/almanac.httparchive.org#contributing" class="btn">Join the team</a>
 {% endblock %}
+
+{% block featured_chapter%}Featured Chapter{% endblock %}
```

Comment:
The 2020 home page doesn't include a featured chapter section yet. Is this needed?

In a separate issue, could we also explore DRY solutions to reduce the duplication of language-specific text blocks across year templates?